### PR TITLE
move NewDefaultMockTokenManagerConfiguration to respective tests to fix golangci issue

### PR DESCRIPTION
**Pull request by dipak-pawar**
Currently we have NewDefaultMockTokenManagerConfiguration in `test/configuration/token_manager_configuration.go` which is using minimock generated mock due to which golangci is failing to compile code and failing with error. See [here](https://golangci.com/r/github.com/fabric8-services/fabric8-common)

Now moving `NewDefaultMockTokenManagerConfiguration` with it's respective tests so that it will compile.

### CODE REVIEW

**Review by alexeykazakov — COMMENTED** [review]
Why golangci can't compile test/configuration/token_manager_configuration.go but has no issues with compiling tests with the same code?

**Comment by dipak-pawar** [comment]
> Why golangci can't compile test/configuration/token_manager_configuration.go but has no issues with compiling tests with the same code?

@alexeykazakov, `test/configuration/token_manager_configuration.go` is importing auto generated code which is not checked in to repository and generated using minimock. If we move same code in tests `go build ./...` doesn't consider `_test.go` while compiling it and hence no issues.

File: token/jwk/fetch_keys_blackbox_test.go [hunk 1]
```diff
@@ -66,3 +67,15 @@ func TestFetchKeys(t *testing.T) {
 	})
 
 }
+
+func defaultMockTokenManagerConfiguration(t *testing.T) *tokensupport.ManagerConfigurationMock {
```
Comment (nurali-techie) [inline token/jwk/fetch_keys_blackbox_test.go:71]: this name looks too verbal `defaultMockTokenManagerConfiguration()` it's feels like java_naming and not very much golang idiomatic.  If not different but can we have `defaultMockTokenMgrConfig()` or something ??

Comment (dipak-pawar) [reply token/jwk/fetch_keys_blackbox_test.go:71]: @nurali-techie I agree, it's too verbal, but I think it's clear from it's name what it is doing

**Review by alexeykazakov — APPROVED** [review]
I still don't like code duplication here... But if you don't see a better way to handle that then I'm fine with that change.

**Pull request merged**